<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QWidget" name="imageWidget" native="true">
      <layout class="QVBoxLayout" name="mplvl"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="controlPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>445</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Spectrometer Controls</string>
       </attribute>
       <widget class="QPushButton" name="stopStreamButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Stop Stream</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startStreamButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Start Stream</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="intField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>3.800000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="intLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Integration time (ms)</string>
        </property>
       </widget>
       <widget class="QLabel" name="backgroundLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Background</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="backShotLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Background shots</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="backShotField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QPushButton" name="takeBackButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Take Background</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QProgressBar" name="backProgress">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="showBackButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Background</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="subtractBackCheck">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Subtract Background</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="postPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>445</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Post Processing</string>
       </attribute>
       <widget class="QPushButton" name="startIntButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Start Integration</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="integrationLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Integration</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="intShotField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="intShotLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Shots to Integrate</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="intProgress">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="showIntButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Integration</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopIntButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Stop Integration</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="avgCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>181</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Average</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <attribute name="label">
        <string>Spectrum Information</string>
       </attribute>
       <widget class="QLabel" name="r_queueLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>r_queue images:</string>
        </property>
       </widget>
       <widget class="QLabel" name="r_queueNum">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="s_queueLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>s_queue images:</string>
        </property>
       </widget>
       <widget class="QLabel" name="s_queueNum">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
